xarray/tests/test_plot.py

```diff
@@ -1675,3 +1671,68 @@ def test_plot_cftime_data_error():
     data = DataArray(data, coords=[np.arange(5)], dims=['x'])
     with raises_regex(NotImplementedError, 'cftime.datetime'):
         data.plot()
+
+
+test_da_list = [DataArray(easy_array((10, ))),
+                DataArray(easy_array((10, 3))),
+                DataArray(easy_array((10, 3, 2)))]
+
+
+@requires_matplotlib
+class TestAxesKwargs(object):
+    @pytest.mark.parametrize('da', test_da_list)
+    @pytest.mark.parametrize('xincrease', [True, False])
+    def test_xincrease_kwarg(self, da, xincrease):
+        plt.clf()
+        da.plot(xincrease=xincrease)
+        assert(plt.gca().xaxis_inverted() == (not xincrease))
```

Comment:
I don't even know if it's worth fixing here, but `assert` is a statement, not a function. So you can write `assert something` rather than `assert(something)`.